packages/table/src/interactions/reorderable.tsx

```diff
@@ -38,6 +39,12 @@ export interface IReorderableProps {
     onSelection: (regions: IRegion[]) => void;
 
     /**
+     * When the user reorders something, this callback is called with the new
+     * focus cell for the newly selected set of regions.
```

Comment:
these docs don't match what I expect from a prop called `onFocus`: i expect a handler for a `focus` event, but you're talking about reordering. can you come up with a more intuitive name?

Reply:
...I'm just cribbing from "onSelection" above -- and that's what the "set my selection" and "set my focus" callbacks are called everywhere. They just filter up to the toplevel onFocus and onSelection behavior. Not sure what the issue is here?

Reply:
@giladgray this is a good point, but it's something we'd have to address more globally, since `onFocus` appears everywhere in `Table` right now. I vote to leave as is in this PR and revisit separately. Tracking in https://github.com/palantir/blueprint/issues/1562.